openapi/paths/kyc-requests.yaml

```
@@ -3,24 +3,37 @@ post:
     - Core
   tags:
     - KYC documents
-  summary: Create a KYC Request
+  summary: Create a KYC request
   operationId: PostKycRequest
   x-sdk-operation-name: create
   description: |-
-    Create a KYC Request.
-    Then, redirect the customer to the `kycGatherer` link.
-    After the customer completes the KYC process, the browser redirects back to the `redirectUrl`.
-    Corresponding webhooks (such as KYC document accepted, KYC document rejected) are sent to webhooks subscribers.
-    When the complete list of documents is received and accepted, the KYC request fulfilled webhook is sent to subscribers.
-    If a `credit-file-proof` is successful with the decision `single-source` or `dual-source`, then corresponding `identity-proof` and `address-proof` documents are not requested.
+    Creates a KYC request.
+
+    The customer is redirected to the `kycGatherer` link.
+    After the customer completes the KYC process,
+    they are redirected back to the `redirectUrl` link.
+
+    Corresponding webhooks, such as: KYC document accepted and KYC document rejected,
+    are sent to the subscribers.
+
+    When the complete list of documents is received and accepted,
+    the KYC request fulfilled webhook is sent to subscribers.
+
+    If a `credit-file-proof` request is successful,
+    it returns a `decision` value of `single-source` or `dual-source`.
+    The corresponding `identity-proof` and `address-proof` documents are not requested.
   requestBody:
     content:
       application/json:
         schema:
           $ref: ../components/schemas/KycRequest.yaml
         examples:
           credit-file-identity-address:
-            summary: Credit file with fallback. In the sandbox environment pass the customer first name `accept` to mark `credit-file-proof` KYC document as `accepted`, and any other first name to reject the document.
+            summary: |-
+              Credit file with fallback.
+              In the sandbox environment, to mark a `credit-file-proof` KYC document as `accepted`,
+              pass the customer's first name to the `accept` field.
+              If an incorrect first name is passed, the document is rejected.
```

This isn't good for a summary. It's good for a description. Summary also doesn't support Markdown.
```suggestion
            summary: Credit file with fallback.
            description: |-
              In the sandbox environment, to mark a `credit-file-proof` KYC document as `accepted`,
              pass the customer's first name to the `accept` field.
              If an incorrect first name is passed, the document is rejected.
```

---

I think I must have misread this one as a description